### ISSUE TITLE
PV default description

### ISSUE THREAD

**Issue by bpulluta** [NREL/REopt.jl#408]
https://github.com/NREL/REopt.jl/blob/f3210466701ba1525792d7c8f167914056e16949/src/core/pv.jl#L52

this needs to be updated to reflect the new defaults.

**Comment by bpulluta** [comment]
Potential fix: !!! note "PV tilt and azimuth"
    If `tilt` is not provided, then it is set to the **20** degrees for both  ground-mount systems  rooftop systems.
    If `azimuth` is not provided, then it is set to 180 if the site is in the northern hemisphere and 0 if in the southern hemisphere.

@adfarth. I looked through the code and didn't see this being applied anywhere: 
> If `tilt` is not provided, then it is set to the absolute value of `Site.latitude` for ground-mount systems. 

If that's the case, I can update it with the fix from above

**Comment by adfarth** [comment]
@bpulluta yes that note is likely a remnant from before the more recent changes. Please go ahead and update with your fix. Thanks!
